pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -673,19 +665,18 @@ def get_local_shells(self, tolerance=None, cluster_by_distances=False, cluster_b
         Returns:
             shells (numpy.ndarray): shell indices
         """
-        allow_ragged = self.allow_ragged
-        if allow_ragged:
-            self.allow_ragged = False
         if tolerance is None:
             tolerance = self._tolerance
+        if mode is None:
```

Comment:
Again, only do this if you see it and like it more, but I've been enjoying `mode = self.mode if mode is None else mode`